src/extension.ts

```diff
@@ -403,6 +449,42 @@ function onConfigurationChange() {
         }
       });
     }
+
+    const serversChanged = !configKeyDeepEquals(connectedModeServersSectionName, oldConfig, newConfig);
+    const bindingChanged = !configKeyDeepEquals(connectedModeProjectSectionName, oldConfig, newConfig);
+
+    if (serversChanged && bindingChanged) {
+      const msg =
+        "SonarLint connected mode server and binding configuration changed, please update server storage and binding.";
+      const action = "Update Now";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 3 times. ([typescript:S1192](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1192))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.vscode%3Asonarlint-vscode&pullRequest=18&issues=AWMmvXt8cd1riAyNLoY8&open=AWMmvXt8cd1riAyNLoY8)

[ISSUE_KEY]: AWMmvXt8cd1riAyNLoY8